rc/tools/git.kak

```diff
@@ -357,6 +358,10 @@ define-command -params 1.. \
                 set-option current grepcmd '$kak_opt_grepcmd'
             }"
             ;;
+        edit)
+            shift
+            echo "edit $@"
```

Comment:
good change, this allows my to drop a few lines from my config.
I'd make it `edit -existing -- $enquoted` (defining enquoted like above).
The `-existing` is debatable.

Reply:
I have modified it according to your comment. I think `-existing` makes sense, so I added it. Effectively, when searching for a specific git file, it is most of the time to quickly open a file that I know exists around, but I don't want to type the whole path.

For a new file, I guess the user would have to revert to a regular `edit` instead of `git edit`. By the way, this can be quickly done by going up in the command history, doing a `Ctrl-a` and `Alt-d` to remove the `git` part of the command and creating the file that was just searched.

Reply:
yes `-existing` is safer for deleted files that haven't yet been deleted in the Git index.
There is an argument that edit is still useful there -- because you could restore the file later and trigger autoreload -- but that requires more presence of mind from the user to not be confused. Probably better to fail early with `-existing`